src/services/facilityService.js

```diff
@@ -0,0 +1,43 @@
+import { Sequelize } from 'sequelize';
+import uuid from 'uuid/v4';
+import db from '../models';
+import stringHelper from '../utils/stringHelper';
+
+/**
+ * @description facility services
+ * @class FacilitiesService
+ */
+export default class FacilityService {
+  /**
+   * @param  {Object} facilityId
+   * @param  {Object} user
+   * @param  {Object} feedback
+   * @returns {Object} NewFeedback
+   */
+  static async feedback(facilityId, user, feedback) {
+    const todayDate = new Date();
+    const facility = await db.Facilities.findByPk(facilityId);
+    if (!facility) {
+      return stringHelper.facilityNotFound;
+    }
```

Comment:
wouldn't it be better to use a ternary operator instead ?

Reply:
They are best used if there is an if-else statement and we only have an if statement only